internal/app/go-filecoin/paymentchannel/testing/fake_init_actor_iface.go

```diff
@@ -0,0 +1,131 @@
+package testing
+
+import (
+	"context"
+	"testing"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	init_ "github.com/filecoin-project/specs-actors/actors/builtin/init"
+	"github.com/filecoin-project/specs-actors/actors/runtime"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+	"github.com/filecoin-project/specs-actors/support/mock"
+	spect "github.com/filecoin-project/specs-actors/support/testing"
+	"github.com/ipfs/go-cid"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/internal/pkg/block"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/vm"
+	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/gas"
+)
+
+// FakeInitActorIface fulfils the MsgSender and MsgWaiter interfaces for a Manager
+// via the specs_actors mock runtime. It executes init.Actor exports directly.
+type FakeInitActorIface struct {
```

Comment:
I debated whether this belongs in a more general place, but right now it's functionality is focused on payment channel .